lib/instrumentation/winston.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const createFormatter = require('@newrelic/winston-enricher/lib/createFormatter')
+const API = require('../../api')
+
+module.exports = function instrument(agent, winston, _, shim) {
+  const config = agent.config
+  if (!config.application_logging || !config.application_logging.enabled) {
```

Comment:
you also need to include `config.application_logging.local_decorating.enabled` I would also wrap this perhaps in some util function check as we will also need it in pino. You can also do all the checks and wrap them in parens and just do `!(<conditions>)`

`if (!(config.application_logging && config.application_logging.enabled && config.application_logging.local_decorating && config.application_logging.local_decorating.enabled))`